robomaker-sample-app-ci/dist/index.js

```diff
@@ -816,9 +816,13 @@ function prepare_sources() {
                 "README*",
                 "roboMakerSettings.json"
             ];
+            const execOptions = {
```

Comment:
Looks like your `index.js` is out of sync with your `aws-robomaker-sample-application-ci.ts`.

Reply:
> Looks like your `index.js` is out of sync with your `aws-robomaker-sample-application-ci.ts`.

Keep it out of sync to test the npm run build....will check it in as final step.

Reply:
Update: index.js is in sync now.